src/main/kotlin/lotto/domain/LottoMachine.kt

```diff
@@ -1,17 +1,37 @@
 package lotto.domain
 
 class LottoMachine {
-    fun createLottos(amount: Amount): List<Lotto> {
-        val number = amount.divide(LOTTO_PRICE).toInt()
-        return List(number) { createAutoLotto() }
+    fun createLottos(
+        amount: Amount,
+        manualLottoNumbers: List<List<Int>> = emptyList(),
```

Comment:
Can other developers understand the meaning of `List<List<Int>>` ?
Is there a way to improve this?

Reply:
I was thinking of two ways on this matter.
1. wrap List<List<Int>> for a more intuitive object
2. change to `List<Lotto>` in the parameter 

However, I chose 1 because I wanted the `LottoMachine` to have all responsibilities for creating lottos. If I had chose 2, I would have had to create lottos in controller. I hope this is what you intended for. If there's anything more, please fix me.